charts/seed-controlplane/charts/kube-apiserver/templates/deployment.yaml

```diff
@@ -340,12 +399,29 @@ spec:
       - name: kube-apiserver-admission-config
         configMap:
           name: kube-apiserver-admission-config
+      {{- if .Values.konnectivityTunnel.enabled }}
+      - name: konnectivity-server-certs
+        secret:
+          secretName: konnectivity-server
+      - name: konnectivity-server-kubeconfig
+        secret:
+          secretName: konnectivity-server-kubeconfig
+      - name: konnectivity-uds
+        emptyDir: {}
+      - name: egress-selection-config
+        configMap:
+          name: kube-apiserver-egress-selector-configuration
+      - name: konnectivity-server-log
+        emptyDir:
+          sizeLimit: 15Mi
```

Comment:
Once the limit is hit the `Pod` is evicted? Isn't it better to also put `--log-file-max-size=12` just to be on the safe side?

Reply:
I though you mentioned that "The log-* flags do nothing" ?